seqr/utils/search/utils.py

```diff
@@ -32,6 +35,94 @@ def get_es_client(timeout=60, **kwargs):
     return elasticsearch.Elasticsearch(**client_kwargs, **kwargs)
 
 
+def ping_search_backend():
+    if not get_es_client(timeout=3, max_retries=0).ping():
+        raise ValueError('No response from elasticsearch ping')
+
+
+def get_elasticsearch_status():
+    client = get_es_client()
+
+    disk_status = {
+        disk['node']: disk for disk in
+        _get_es_meta(client, 'allocation', ['node', 'shards', 'disk.avail', 'disk.used', 'disk.percent'])
+    }
+
+    node_stats = {}
+    for node in _get_es_meta(client, 'nodes', ['name', 'heap.percent']):
+        if node['name'] in disk_status:
+            disk_status[node.pop('name')].update(node)
+        else:
+            node_stats[node['name']] = node
+
+    indices, seqr_index_projects = _get_es_indices(client)
+
+    errors = ['{} does not exist and is used by project(s) {}'.format(
+        index, ', '.join(['{} ({} samples)'.format(p.name, len(indivs)) for p, indivs in project_individuals.items()])
+    ) for index, project_individuals in seqr_index_projects.items() if project_individuals]
+
+    return {
+        'indices': indices,
+        'diskStats': list(disk_status.values()),
+        'nodeStats': list(node_stats.values()),
+        'errors': errors,
+    }
+
+
+def delete_es_index(index):
```

Comment:
Similarly, deleting an ES index should be under the `elasticsearch` folder. It is better to have a general DB deletion function under the `search` folder, which calls the `delete_es_index` from the Elasticsearch folder.